types/types.d.ts

```diff
@@ -122,6 +122,12 @@ declare namespace FastifySessionPlugin {
 
     /** Function used to generate new session IDs. Defaults to uid(24). */
     idGenerator?(request?: Fastify.FastifyRequest): string;
+
+    /**
+     * Prefixes all cookie values. Run with "s:" to be be compatible with express-session.
+     * Defaults to false.
```

Comment:
```suggestion
     * @default undefined
```

Reply:
Oops! I actually made it default to `''`. That okay?

![image](https://user-images.githubusercontent.com/736775/184164350-16b90eb7-2e11-48d2-858a-8ea14eedb3a2.png)

Reply:
```suggestion
     * Defaults to ""
```